### PR TITLE
Repo cleaned and first rule added

### PR DESCRIPTION
**Pull request by aniagut**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,143 @@
...
+- :white_check_mark: Image format (lighter-image-files, ec0lint)
+- :white_check_mark: Light libraries - jQuery Ajax (no-ajax, ec0lint)
+- :white_check_mark: Light libraries - jQuery Ajax events (no-ajax-events, ec0lint)
+- :white_check_mark: Font-display (require-font-display, ec0lint-style)
+
+:hammer: v2.1.0 - December - 25 rule implementations, 15 unique rules
+
+- :hammer: Video format (lighter-video-files, ec0lint-plugin-react)
+- :hammer: Video format (lighter-video-files, ec0lint)
+- :hammer: Plugin HTML (ec0lint-plugin-html)
```
Comment (malwinq) [inline README.md]: Pls mark your changes with blue box

File: docs/rules/lazy-load.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+For example, images or videos further down on a website are only loaded if the user scrolls down. 
+With lazy loading, less data needs to be transferred and thus less energy is consumed.
+
+### CO2 reduction
+
+By using lazy loading, we provide that no unnecessary images and videos will be transferred, 
+so for the file that won't be displayed by website user, we limit CO2 emissions connected with loading this 
+file to 0g.
+By multiplying the file size by the end-user traffic (0.81 kWh / 1000 Mb) and by the energy emissions (442 g/kWh) , the carbon footprint of the average image file (3 MB) sums up to 8.59 g. 
+So, by lazy loading this file we could get 8.59g CO2 reduction (100% less CO2).
```
Comment (malwinq) [inline docs/rules/lazy-load.md]: Update after the documentation is approved

File: lib/rules/lazy-load.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+      description: "Suggest to use lazy load",
+      category: RULE_CATEGORY.BEST_PRACTICE,
+      recommended: false,
+    },
+
+    fixable: null,
+    schema: [],
+    messages: {
+      [MESSAGE_IDS.MISSING_LAZY_LOADING]:
+        "You should consider using lazy-loading for this file.",
```
Comment (malwinq) [inline lib/rules/lazy-load.js]: Lets discuss the log message to be consistent with the new style

File: lib/rules/lazy-load.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+          return;
+        }
+
+        if (
+          (isImageTag && !isImageLazyLoaded(node)) ||
+          (isVideoTag && !isVideoLazyLoaded(node))
+        ) {
+          context.report({
+            loc: node.loc,
+            messageId: MESSAGE_IDS.MISSING_LAZY_LOADING,
```
Comment (malwinq) [inline lib/rules/lazy-load.js]: Do we tell a user what CO2 reduction it is? Is there a way to calculate it?

File: package.json [hunk 5]
```diff
@@ -1,37 +1,57 @@
 {
-  "name": "@html-eslint/html-eslint",
-  "version": "0.4.0",
-  "description": "",
-  "main": "index.js",
+  "name": "html-ec0lint",
+  "version": "0.0.1",
+  "description": "Ec0lint plugin for html",
+  "author": "ec0lint",
+  "homepage": "https://github.com/ec0lint#readme",
```
Comment (malwinq) [inline package.json]: Let's use ec0lint landing page

File: README.md (hunk 1 above)
Comment (aniagut) [reply README.md]: Done! @malwinq

File: lib/rules/lazy-load.js (hunk 3 above)
Comment (malwinq) [reply lib/rules/lazy-load.js]: Lazy loading can be used for this file / {file_name}

File: package.json (hunk 5 above)
Comment (aniagut) [reply package.json]: Done

File: lib/rules/lazy-load.js (hunk 4 above)
Comment (aniagut) [reply lib/rules/lazy-load.js]: Will be resolved by creating new CO2 module for this plugin

File: lib/rules/lazy-load.js (hunk 3 above)
Comment (aniagut) [reply lib/rules/lazy-load.js]: Updated

File: docs/rules/lazy-load.md (hunk 2 above)
Comment (aniagut) [reply docs/rules/lazy-load.md]: Updated to current version

**Review by malwinq — APPROVED** [review]
LGTM, thank you @aniagut

**Pull request merged**